libs/app-loader/src/index.ts

```diff
@@ -315,6 +315,11 @@ export default class AppLoader {
     if (!loginData?.id) {
       return;
     }
+
+    // initialise browser notifications
+    const sdk = getSDK();
+    await sdk.api.profile.initNotificationsStream();
```

Comment:
@kenshyx added initialization call here after login is made. I think thats all for the `DEV-402` now.